test/e2e_node/inode_eviction_test.go

```diff
@@ -206,6 +206,12 @@ func runEvictionTest(f *framework.Framework, testCondition string, podTestSpecs
 		})
 
 		AfterEach(func() {
+			By("deleting pods")
```

Comment:
If you delete the pod here, we won't know whether it is the eviction relieves the disk pressure, or your deletion.